public/data/JoBaHP.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Jovan R.",
+  "type": "personal",
+  "bio": "Full Stack Web Developer!",
+  "avatar": "https://github.com/JoBaHP.png",
+  "links": [
+    {
+      "name": "My business website",
+      "url": "http://build-up.us",
+      "icon": "link"
+    },
+    {
+      "name": "Personal portfolio website",
+      "url": "https://jovan-portfolio.live/",
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/JoBaHP",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/https://twitter.com/joccoo",
```

Comment:
Broken link, seems like you have pasted the original link after typing the half link manually.

Small fix! 🛠

```suggestion
      "url": "https://twitter.com/joccoo",
```